data/questionnaires/brain-development-and-how-children-learn.yml

```diff
@@ -114,13 +114,15 @@ brain-development-and-how-children-learn:
         label: |
           Which of these is an early years approach based on the work of Piaget? (Select all that apply) 
         assessment_summary: |
-          Piaget did not believe that babies were born with a fixed intellectual ability. Instead, he said that a child’s interaction with their environment, combined with their growing brain would shape their developing their cognitive skills. 
+          Piaget did not believe that babies were born with a fixed intellectual ability. Instead, he said that a child’s interaction with the environment, combined with their growing brain would shape how their cognitive skills develop. 
         assessment_fail_summary: |
           Piaget’s work highlighted the importance of:
+          
           - a supportive environment to promote development 
           - social interaction and peer teaching to promote development 
           - opportunities to test inconsistencies to promote development 
-          Piaget did not believe that babies were born with a fixed intellectual ability. Instead, he said that a child’s interaction with their environment, combined with their growing brain would shape their developing their cognitive skills. 
```

Comment:
if we're getting rid of this row, I feel like the row below starting 'Instead,' doesn't make sense?